og_access/og_access.install

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Drupal\og\Og;
+
+/**
+ * @file
+ * Install/update/uninstall hook implementations.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function og_access_install() {
+  /** @var \Drupal\og\GroupTypeManager $group_type_manager */
+  $group_type_manager = \Drupal::service('og.group_type_manager');
+  $groups = $group_type_manager->getAllGroupBundles('node');
+
+  foreach ($groups as $group) {
+    Og::createField(OG_ACCESS_FIELD, 'node', $group);
```

Comment:
This is not happening in D7 on install.

I guess it could follow the OG pattern and extend the node type form. It gets complicated otherwise as one has to make the distinction between regular node types, group and group content if adding via the Field UI